public/app/features/alerting/unified/components/amroutes/AmRoutesExpandedForm.tsx

```diff
@@ -56,73 +58,85 @@ export const AmRoutesExpandedForm: FC<AmRoutesExpandedFormProps> = ({ onCancel,
           <FieldArray name="object_matchers" control={control}>
             {({ fields, append, remove }) => (
               <>
-                <div>Matching labels</div>
```

Comment:
This diff looks larger than it ought to be – I've just wrapped the matchers in a `<VerticalGroup>`